Dockerfile.dev

```diff
@@ -0,0 +1,10 @@
+FROM python:3.6
```

Comment:
consider https://github.com/elifesciences/elife-base-images/blob/master/Dockerfile.python_3.6#L1 which may be smaller in size, since it doesn't seem you need particular OS packages to be installed

Reply:
I meant `python:3.6.4-slim-jessie` which is an official one, also the eLife base image can work but it's not mandatory

Reply:
OK, I'll try the official python image first.

Reply:
After testing, it seems that the full python is required to install dependencies.